libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -406,6 +422,9 @@ class AC_PosControl
     Vector3f    _accel_target;          // acceleration target in NEU cm/s/s
     Vector3f    _limit_vector;          // the direction that the position controller is limited, zero when not limited
     Vector2f    _vehicle_horiz_vel;     // velocity to use if _flags.vehicle_horiz_vel_override is set
+    float       _pos_offset_z;            // vertical position offset in NEU cm from home
```

Comment:
nitpick: there's a small indenting problem in the comments on these three new lines.